OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnPrerequisite.cs

```diff
@@ -26,19 +26,23 @@ public class GrantConditionOnPrerequisiteInfo : ITraitInfo
 		[Desc("List of required prerequisites.")]
 		public readonly string[] Prerequisites = { };
 
+		[Desc("Defines whether the condition will be granted to the new owner",
+			"(if the required prerequisites are present)",
+			"instead of being locked to the old owner.")]
+		public readonly bool ResetOnOwnerChange = true;
```

Comment:
Keeping a reference to the wrong player's tech tree is a bug, so why would anybody want to disable this?